scripts/agency_ingestion/run.py

```diff
@@ -180,11 +180,11 @@ def get_data(self):
 		# summarise data by keeping last date
 		last_seen_date = {}
 		for row in data:
-			if row["DATE"] > last_seen_date.get(row["COUNTRY"], ""):
-				last_seen_date[row["COUNTRY"]] = row["DATE"]
+			if row.get("DATE", "") > last_seen_date.get(row.get("COUNTRY"), ""):
+				last_seen_date[row.get("COUNTRY")] = row.get("DATE")
 		self.data = [
 			row for row in data if
-			last_seen_date[row["COUNTRY"]] == row["DATE"]
+			last_seen_date.get(row.get("COUNTRY")) == row.get("DATE")
```

Comment:
The inner `get` is safe, the outer `get` is not

Reply:
Any hashable type (including None) is safe to pass to `get()`, there was another possible bug where both `get()` would default to None and test for equality, so changed one of them to return `""`